src/peergos/server/tests/simulation/Simulator.java

```diff
@@ -212,61 +344,102 @@ public void run() {
 
         LOG.info("Running file-system verification");
         boolean isVerified = verify();
-        LOG.info("System verified =  "+ isVerified);
+        LOG.info("System verified =  " + isVerified);
+    }
+
+    private boolean verifySharingPermissions(String user, Path path) {
```

Comment:
This seems to be verifying the sharer's view of the permissions, but not what the sharee can actually read?

Reply:
That is the case - will update to actually read/write now.